src/honkalculate.template.html

```diff
@@ -886,7 +887,7 @@
         </fieldset>
     </div>
     <div>
-        <fieldset class="panel holder-0">
+        <fieldset class="panel" id="holder-0">
```

Comment:
I change the code in `honkalculate_controls.js` to accomodate the fact that it is an ID instead of a class.

I did delete the CSS code that was associated with `holder-0` since I had commented out all of that CSS code and it wasn't necessary anymore.